app/components/modal.tsx

```diff
@@ -37,7 +37,8 @@ export function Modal({
     "z-20",
     "shadow-5",
     "transition-transform",
```

Comment:
duration-0 is the default, duration-500 is the motion-safe variant. Normally a default wouldn't have to be declared, but duration happens to default to 150 with Tailwind.